packages/core/tests/jsonld.ts

```diff
@@ -38,13 +41,15 @@ export const getJsonLdModules = ({
     }),
     proofs: new ProofsModule({
       autoAcceptProofs,
+      proofProtocols: [new V2ProofProtocol({ proofFormats: [new PresentationExchangeProofFormatService()] })],
     }),
     cache: new CacheModule({
       cache: new InMemoryLruCache({ limit: 100 }),
     }),
     indySdk: new IndySdkModule({
       indySdk,
     }),
+    pex: new PresentationExchangeModule(),
```

Comment:
Is this registered by default? As it's in core, I think we should register it by default